1Password/homebrew-tap
# Update cask to use zip and not require sudo

## Context
Since the latest desktop app version, the CLI can now be installed outside of `/usr/local/bin`. 

## Problem
Our brew cask requires the root password because it's using the pkg installer and places the binary in `/usr/local/bin`

## Solution
This MR changes our cask formula to download the `op` Darwin zip for latest version, for either arm64 or amd64, and place it in the mainstream `/opt/homebrew/bin` dir.

## Thought process
On MacOS, only 2 architectures are supported: arm64 or amd64 (x86_64). `/opt/homebrew/bin` has been chosen because that's where `brew` installs its packages and it requires no permission to be granted in the process.

zap stanza deletes the `.op` config file:
> The zap stanza describes a more complete uninstallation of files associated with a cask. The zap procedures will never be performed by default, but only if the user uses --zap on uninstall.



## How to test
You can test that this runs okay by uninstalling your `op`, checking out this branch and running:  `HOMEBREW_NO_AUTO_UPDATE=1 brew install -v --build-from-source --HEAD Casks/1password-cli.rb`. 

Then run `which op` to check that it's in `/opt/homebrew/bin/op`.
